modules/ROOT/partials/release-notes/deleted.adoc

```diff
@@ -34,13 +34,13 @@ https://www.tiny.cloud/docs-4x/demo/custom-toolbar-listbox/[Listbox] is no longe
 
 The following plugins from TinyMCE 4.x do not require height or width options to be specified in TinyMCE 5.0:
 
-* link:{baseurl}/plugins/opensource/code/[Code]
-* link:{baseurl}/plugins/opensource/codesample/[Codesample]
-* link:{baseurl}/plugins/opensource/preview/[Preview]
-* link:{baseurl}/plugins/opensource/template/[Template]
+* xref:plugins/opensource/code.adoc[Code]
+* xref:plugins/opensource/codesample.adoc[Codesample]
+* xref:plugins/opensource/preview.adoc[Preview]
+* xref:plugins/opensource/template.adoc[Template]
 
-Refer to the link:{baseurl}/migration-from-4x/[Migration Guide], for more information.
+Refer to the xref:migration-from-4x.adoc[Migration Guide], for more information.
 
 === Other removed component configuration settings
 
-Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. Please see this link:{baseurl}/migration-from-4x/#otherremovedcomponentconfigurationsettings[section] of the Migration Guide for more information.
+Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. Please see this xref:migration-from-4x.adoc#otherremovedcomponentconfigurationsettings[section] of the Migration Guide for more information.
```

Comment:
This xref doesn't resolve as the `otherremovedcomponentconfigurationsettings` id doesn't exist on the relevant header in `migration-from-4x.adoc`. However, it also is broken on production as well. As such this probably needs changing to:
```suggestion
Some UI configurations in TinyMCE 5.0 have changed due to an updated UI engine. Please see this xref:migration-from-4x.adoc#removeduiconfigurationsettings[section] of the Migration Guide for more information.
```
and then the id would need to be added to the heading. e.g:
```
[[removeduiconfigurationsettings]]
=== Removed UI configuration settings
```

Reply:
done